backend/tet/events/transformations.py

```diff
@@ -25,11 +25,23 @@ def _new_from(obj, keys):
     return new
 
 
+def _shorten_description(descobj):
+    shortened_obj = {}
+    for lang in descobj.keys():
+        desc = descobj[lang]
+        if len(desc) <= 125:
+            shortened_obj[lang] = desc
+        else:
+            shortened_obj[lang] = f"{desc[:125]}\u2026"
```

Comment:
It would be beneficial for understandability to add something to tell what the \u2026 character is, e.g.:

import unicodedata
...
shortened_obj[lang] = f"{desc[:125]}{unicodedata.lookup('Horizontal ellipsis')}"
or
ellipsis = "\u2026"
shortened_obj[lang] = f"{desc[:125]}{ellipsis}"
or
ellipsis = unicodedata.lookup('Horizontal ellipsis')
shortened_obj[lang] = f"{desc[:125]}{ellipsis}"
or
shortened_obj[lang] = f"{desc[:125]}\u2026"  # \u2026 = …